frontend/packages/dev-console/src/components/import/__tests__/import-submit-utils.spec.ts

```diff
@@ -0,0 +1,151 @@
+import * as _ from 'lodash';
+import * as k8s from '@console/internal/module/k8s';
+import {
+  DeploymentConfigModel,
+  DeploymentModel,
+  ImageStreamModel,
+  ServiceModel,
+  RouteModel,
+  BuildConfigModel,
+  SecretModel,
+} from '@console/internal/models';
+import { Resources } from '../import-types';
+import * as submitUtils from '../import-submit-utils';
+import { defaultData, nodeJsBuilderImage as buildImage } from './import-submit-utils-data';
+
+const { createDeployment, createResources } = submitUtils;
+
+describe('Import Submit Utils', () => {
+  describe('createDeployment tests', () => {
+    beforeAll(() => {
+      jest
+        .spyOn(k8s, 'k8sCreate')
+        .mockImplementation((model, data, dryRun) => ({ model, data, dryRun }));
```

Comment:
Same here, shouldn't this be returning a wrapped `Promise`?

Reply:
changed